packages/js/src/utils/swc/compile-swc.ts

```diff
@@ -0,0 +1,64 @@
+import { TypeScriptCompilationOptions } from '@nrwl/workspace/src/utilities/typescript/compilation';
+import { exec, execSync } from 'child_process';
+import { normalizeTsCompilationOptions } from '../normalize-ts-compilation-options';
+
+export async function compileSwc(
+  tsCompilationOptions: TypeScriptCompilationOptions,
+  postCompilationCallback: () => void | Promise<void>
+): Promise<{ success: boolean }> {
+  const normalizedOptions = normalizeTsCompilationOptions(tsCompilationOptions);
+
+  console.log(`Compiling with SWC for ${normalizedOptions.projectName}...`);
```

Comment:
Is this for debugging, or do we want to log? Use `logger` from `@nrwl/devkit` if it's the latter.

Reply:
Fixed